coding-guidelines/coding-guidelines.md

```diff
@@ -0,0 +1,243 @@
+# Coding Guidelines
+
+## Line length
+One Line in JavaScript should have a maximal length of 80 characters.
```

Comment:
"Line" should be lowercase I think or is this on purpose?

Also I would be more strict and say "must" instead of "should".

Maybe:
`One line must not exceed the limit of 80 characters.`

And other lines that are not JavaScript have this limit too.